default.nix

```diff
@@ -11,9 +11,9 @@ let
   inherit (specs.this) deps;
 
   # Import deploy scripts from dss-deploy
-  dss-deploy' = if (dss-deploy != null)
-    then dss-deploy
-    else import deps.dss-deploy.src' {};
+  dss-deploy' = if isNull dss-deploy
+    then import deps.dss-deploy.src' { inherit pkgs; }
```

Comment:
Did it work without the `inherit`?

Reply:
Good catch, I missed that I added that.

So with the `inherit pkgs` `dss-deploy` will use the same `pkgs` as `tdds` which might not be desirable. `dss-deploy` should probably make it's own choice of which `pkgs` version to use.

Reply:
Do you want me to apply the suggestion? If so, I think it would be nice to clean up the branch afterwards (squash my commit), so it doesn’t make much sense for me to do it :)

Reply:
BTW, when would we want them to have different `pkgs`?

Reply:
For example when we need `dss-deploy` to run a certain set of `pkgs` without updating `dss-deploy` it self. But then we need to do some integration testing to make sure `dss-deploy` works with another set of `pkgs` then intended.

Also injecting `pkgs` could be a pattern when developing, so that when testing locally you can propagate the `pkgs` you have locally. This is now solved by being able to inject the entire `dss-deploy` as an argument to the expression.